indico/modules/rb/client/js/common/bookings/BookingDetails.jsx

```diff
@@ -526,6 +543,22 @@ class BookingDetails extends React.Component {
               <Translate>Booking Details</Translate>
             </span>
             <span styleName="booking-status">{this.renderBookingStatus()}</span>
+            <span styleName="clipboard-copy">
+              <Popup
+                on="click"
+                position="bottom center"
+                trigger={
+                  <Icon
+                    name="linkify"
+                    onClick={async () => await navigator.clipboard.writeText(externalDetailsURL)}
```

Comment:
you do not need to await this since you aren't doing anything after copying.

however, i think the logic here is incorrect: the popup should only open after having awaited the writeText operation, in case it fails